force-app/main/default/lwc/barcodeScannerExample/barcodeScannerExample.js

```diff
@@ -0,0 +1,106 @@
+// barcodeScannerExample.js
+import { LightningElement } from 'lwc';
+import { NavigationMixin } from 'lightning/navigation';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
```

Comment:
We generally try to avoid using toast notifications to display error messages for the sake of UX but I don't know if this best practice applies to mobile. Any clue @albarivas?
@tessaSAC for reference, take a look at what @albarivas did with the `c-error-panel` component in `c-property-location`. This is our common error reporting pattern.

Reply:
I assume that you mean [these examples](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/use_barcodescanner_example) from the LWC docs?
Our team doesn't own these examples so the code doesn't adhere to the same conventions that we use for sample apps.
Perhaps we can ask the doc team to take a look at our example once we're done with your PR.

Reply:
Yep, TIL! After discussing with @albarivas, it sounds like the toast is the way to go for errors on a mobile-only experience, so that users can also see the result in their notification drawer.